homeassistant/components/schlage/sensor.py

```diff
@@ -66,3 +56,35 @@ def _handle_coordinator_update(self) -> None:
         """Handle updated data from the coordinator."""
         self._attr_native_value = getattr(self._lock, self.entity_description.key)
         return super()._handle_coordinator_update()
+
+
+class SchlageBatterySensor(SchlageSensor):
+    """Schlage battery sensor entity."""
+
+
+class SchlageDurationSensor(SchlageSensor):
+    """Schlage duration sensor entity."""
```

Comment:
```suggestion
```

Can be removed as there is no difference between them, so we can use directly the base class

Reply:
Wasn't sure if that would be a breaking change